src/lib.rs

```diff
@@ -24,7 +24,7 @@
 
 #![no_std]
 // Catch documentation errors caused by code changes.
-#![deny(intra_doc_link_resolution_failure)]
+#![deny(broken_intra_doc_links)]
```

Comment:
GH actions doesn't seem to understant that lint. It's fine if it works anyway

Reply:
They renamed this recently to `broken_intra_doc_links`

https://github.com/rust-lang/rust/pull/74926

With a updated nightly you should receive a warning/error if using `intra_doc_link_resolution_failure`

Reply:
https://github.com/rust-lang/cargo/pull/8576
https://github.com/rust-lang/rust/pull/74926